### ISSUE TITLE
Errors running e2e tests

### ISSUE THREAD

**Issue by m0cs** [fullcalendar/fullcalendar-angular#368]
I try to run e2e tests on **master** branch with tag **5.7.0**: _npm run e2e_, but, they crash by timeout.

Steps:
1st - checkout master
2nd - npm install
3rd - npm run e2e

node: 10.21.0
angular: 11.0.2

<img width="1059" alt="Screen Shot 2021-05-25 at 11 44 30" src="https://user-images.githubusercontent.com/7376855/119518097-992e9e00-bd4e-11eb-8c42-772ed9ee831e.png">

**Comment by acerix** [comment]
I can confirm the same issue on Linux and Yarn.

```
$ yarn e2e
yarn run v1.22.10
$ ng e2e
...
✔ Compiled successfully.
[17:35:13] I/launcher - Running 1 instances of WebDriver
[17:35:13] I/direct - Using ChromeDriver directly...
WARN: jasmine-spec-reporter 'displayStacktrace' option supports value ('none', 'raw', 'pretty'), default to 'none'

Jasmine started

  workspace-project App
    ✗ should display calendar component
      - Failed: script timeout
        (Session info: chrome=90.0.4430.212)
        (Driver info: chromedriver=90.0.4430.24 (4c6d850f087da467d926e8eddb76550aed655991-refs/branch-heads/4430@{#429}),platform=Linux 5.12.6-zen1-1-zen x86_64)
    ✗ should display header
      - Failed: script timeout
        (Session info: chrome=90.0.4430.212)
        (Driver info: chromedriver=90.0.4430.24 (4c6d850f087da467d926e8eddb76550aed655991-refs/branch-heads/4430@{#429}),platform=Linux 5.12.6-zen1-1-zen x86_64)
    ✗ should display month button
      - Failed: script timeout
        (Session info: chrome=90.0.4430.212)
        (Driver info: chromedriver=90.0.4430.24 (4c6d850f087da467d926e8eddb76550aed655991-refs/branch-heads/4430@{#429}),platform=Linux 5.12.6-zen1-1-zen x86_64)
    ✗ should update header
      - Failed: script timeout
        (Session info: chrome=90.0.4430.212)
        (Driver info: chromedriver=90.0.4430.24 (4c6d850f087da467d926e8eddb76550aed655991-refs/branch-heads/4430@{#429}),platform=Linux 5.12.6-zen1-1-zen x86_64)

**************************************************
*                    Failures                    *
**************************************************

1) workspace-project App should display calendar component
  - Failed: script timeout
    (Session info: chrome=90.0.4430.212)
    (Driver info: chromedriver=90.0.4430.24 (4c6d850f087da467d926e8eddb76550aed655991-refs/branch-heads/4430@{#429}),platform=Linux 5.12.6-zen1-1-zen x86_64)

2) workspace-project App should display header
  - Failed: script timeout
    (Session info: chrome=90.0.4430.212)
    (Driver info: chromedriver=90.0.4430.24 (4c6d850f087da467d926e8eddb76550aed655991-refs/branch-heads/4430@{#429}),platform=Linux 5.12.6-zen1-1-zen x86_64)

3) workspace-project App should display month button
  - Failed: script timeout
    (Session info: chrome=90.0.4430.212)
    (Driver info: chromedriver=90.0.4430.24 (4c6d850f087da467d926e8eddb76550aed655991-refs/branch-heads/4430@{#429}),platform=Linux 5.12.6-zen1-1-zen x86_64)

4) workspace-project App should update header
  - Failed: script timeout
    (Session info: chrome=90.0.4430.212)
    (Driver info: chromedriver=90.0.4430.24 (4c6d850f087da467d926e8eddb76550aed655991-refs/branch-heads/4430@{#429}),platform=Linux 5.12.6-zen1-1-zen x86_64)

Executed 4 of 4 specs (4 FAILED) in 45 secs.
[17:36:01] I/launcher - 0 instance(s) of WebDriver still running
[17:36:01] I/launcher - chrome #01 failed 4 test(s)
[17:36:01] I/launcher - overall: 4 failed spec(s)
[17:36:01] E/launcher - Process exited with error code 1
error Command failed with exit code 1.
```

**Comment by Timebutt** [comment]
I fixed these issues in my Angular 13 upgrade branch #386.

**Comment by arshaw** [comment]
Also fixed in the [v6 branch](https://github.com/fullcalendar/fullcalendar-scheduler/tree/v6)

**Comment by arshaw** [comment]
Released in [v6.0.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.0.0)
